src/constants/addresses/index.ts

```diff
@@ -11,6 +11,7 @@ import { TOKENS as TOKENS_TESTNET  } from './testnet/tokens';
 
 export const Addresses = {
   genesis: '0x0000000000000000000000000000000000000000',
+  bnb: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
```

Comment:
Is there a reason you are using this address for BNB? Because BNB is not a BEP-20 token and as such does not have an address